src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/UISettings.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Windows;
+using System.Windows.Interop;
+using System.Windows.Media;
+using MS.Internal.PresentationCore.WindowsRuntime;
+
+namespace MS.Internal.WindowsRuntime
+{
+    namespace Windows.UI.ViewManagement
+    {
+        internal class UISettings : IDisposable
+        {
+            private readonly bool _isSupported;
+
+            private UISettingsRCW.IUISettings3 _uisettings;
+
+            private static Color _fallbackAccentColor = Color.FromArgb(0xff, 0x00, 0x78, 0xd4);
+
+            private Color _accentColor, _accentLight1, _accentLight2, _accentLight3;
+            private Color _accentDark1, _accentDark2, _accentDark3;
+
+            private bool _useFallbackColor = false;
+
+            internal UISettings()
+            {
+                _isSupported = true;
+
+                try
+                {
+                    _uisettings = GetWinRTInstance() as UISettingsRCW.IUISettings3;
+                }
+                catch (COMException)
+                {
+                }
+
+                if (_uisettings == null)
+                {
+                    _isSupported = false;
+                }
+            }
+
+            internal bool GetColorValue(UISettingsRCW.UIColorType desiredColor, out Color color)
```

Comment:
Since this fn doesn't return `Color`, we should better name it as `TryGetColorValue`

Reply:
Done